sound/soc/intel/boards/sof_sdw.c

```diff
@@ -569,6 +569,7 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.direction = {true, true},
 				.dai_name = "rt700-aif1",
 				.dai_type = SOF_SDW_DAI_TYPE_JACK,
+				.dailink = {SDW_JACK_IN_DAI_ID, SDW_JACK_OUT_DAI_ID},
```

Comment:
is this correct?

the array is listed by direction, so the playback is first, then capture, but this shows the opposite order?

Not sure what I am missing.

Reply:
> is this correct?
> 
> the array is listed by direction, so the playback is first, then capture, but this shows the opposite order?
> 
> Not sure what I am missing.

Thank you @plbossart I fixed it now. It worked when I tested it because the defines were
#define SDW_JACK_IN_DAI_ID 0
#define SDW_JACK_OUT_DAI_ID 1